dwebsummit/dwebsummit_frontend/static/css/_desktop.scss

```diff
@@ -81,21 +81,19 @@
     margin-top: 0rem;
     margin-bottom: 2rem;
   }
-  section.sponsors {
+  #sponsors {
```

Comment:
Suggestion: As a general rule, we avoid using “id” attributes unless we need to. I don’t *think* it’s necessary here. Applies below too.